release_notes/topics/4_1_0_final.adoc

```diff
@@ -1,3 +1,8 @@
 = Making Spring Boot 2 the default starter
 
-Starting with release 4.1, the Spring Boot starter will be based on the Spring Boot 2 adapter. If you are using an older Spring Boot version, the keycloak-legacy-spring-boot-starter is available.
\ No newline at end of file
+Starting with release 4.1, the Spring Boot starter will be based on the Spring Boot 2 adapter. If you are using an older Spring Boot version, the keycloak-legacy-spring-boot-starter is available.
```

Comment:
This is the wrong place. You're adding to 4.1 release notes, while we're doing 4.3.

Reply:
Actually it's in since 4.2 but I wonder why this change is in this PR.